channeldb/channel.go

```diff
@@ -394,19 +394,11 @@ func (c ChannelType) IsTaproot() bool {
 	return c&SimpleTaprootFeatureBit == SimpleTaprootFeatureBit
 }
 
-// ChannelConstraints represents a set of constraints meant to allow a node to
-// limit their exposure, enact flow control and ensure that all HTLCs are
-// economically relevant. This struct will be mirrored for both sides of the
-// channel, as each side will enforce various constraints that MUST be adhered
-// to for the life time of the channel. The parameters for each of these
-// constraints are static for the duration of the channel, meaning the channel
-// must be torn down for them to change.
-type ChannelConstraints struct {
-	// DustLimit is the threshold (in satoshis) below which any outputs
-	// should be trimmed. When an output is trimmed, it isn't materialized
-	// as an actual output, but is instead burned to miner's fees.
-	DustLimit btcutil.Amount
-
+// ChannelStateSpaceBounds are the parameters from OpenChannel and AcceptChannel
+// that are responsible for providing bounds on the state space of the abstract
+// channel state. These values must be remembered for normal channl operation
+// but they do not impact how we compute the commitment transactions themselves.
+type ChannelStateSpaceBounds struct {
```

Comment:
nit: I propose simpler naming: `ChannelStateBounds` and `CommitmentParams`. Subjective, but imo simpler names can make sense for readability.

Reply:
Yeah I opted for precise, but couldn't settle on something concise. I like your suggestions. They have been incorporated.